src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -200,6 +200,43 @@ WorkspaceServicePositionColumnOutOfRange(int line) = Position is outside of colu
 WorkspaceServiceBufferPositionOutOfOrder(int sLine, int sCol, int eLine, int eCol) = Start position ({0}, {1}) must come before or be equal to the end position ({2}, {3})
 
 ############################################################################
+# Edit Data Service
+
+EditDataSessionNotFound = Edit session does not exist.
+
+EditDataUnsupportedObjectType(string typeName) = Database object {0} cannot be used for editing.
+
+EditDataQueryNotCompleted = Query has not completed execution
```

Comment:
A lot of these might be weird when shown in the front-end UI. In what scenarios will they be used? Can we get a PM to review them? I care more about ones that are likely to show up a lot vs. diagnostic / edge case error messages